db/CSVConnector.java

```diff
@@ -101,6 +101,12 @@ private String getNextId(String _table) {
    
     @Override
     public Boolean updateObject(Map<String, String> _keyValuePairs, String _uuid, String _table) {
+        /*if (_table links to users.csv)
+            update password
+          if (_table links to scores.csv)
+            update scores from Scores.java
+        */
```

Comment:
This logic isn't totally correct. updateObject() should take whatever _keyValuePairs it gets, and just shove them into whatever _table it gets, where the uuid matches _uuid. In the case of updating the password, the only keyValuePair it would get would be :
`{password: "myNewPasssword123"}` .  And then it would update the password.